_plugins/elixir_school.rb

```diff
@@ -137,15 +137,20 @@ def contents(site, lang)
         chapters.each_with_index do |chapter_name, index|
           if chapter = get_chapter(site, lang, section, chapter_name)
             chapter['chapter_number'] = index + 1
-            current = [section, chapter_name]
+            current = {
+              'section' => section,
+              'chapter' => chapter_name,
+              'url' => chapter['url'],
+              'title' => chapter['title']
+            }
```

Comment:
I initially took this approach, but then i discarted by a two element array, because we are going to be duplicating information way too much. Since you have `site['tree']` there is no need to store `url` and `title` given `lang`, `chapter` and `section`.